src/Fixer/Alias/NoMixedEchoPrintFixer.php

```diff
@@ -42,14 +47,14 @@
     public function configure(array $configuration = null)
     {
         if (null === $configuration) {
-            $configuration = self::$defaultConfig;
-        } else {
-            if (1 !== count($configuration) || !array_key_exists('use', $configuration) || !in_array($configuration['use'], array('print', 'echo'), true)) {
-                throw new InvalidFixerConfigurationException($this->getName(), sprintf(
-                    'Expected array of element "use" with value "echo" or "print", got "%s".',
-                    var_export($configuration, true)
-                ));
-            }
+            $configuration = self::$defaultConfiguration;
+        }
+
+        if (1 !== count($configuration) || !array_key_exists('use', $configuration) || !in_array($configuration['use'], array('print', 'echo'), true)) {
```

Comment:
I'm not sure about some of the differences between different implementations of `configure()`, when no configuration is provided:

* sometimes we just set `$configuration` to a default, then still validate it, and eventually assign it to the instance field
* sometimes we just set the instance field, without any validation of the default configuration

Both have pro and cons (faster without extra validation, error prone without extra validation).

Does anybody have any thoughts to that?